src/main/webapp/app/exercises/programming/manage/instructions-editor/analysis/programming-exercise-instruction-analysis.service.ts

```diff
@@ -113,19 +107,19 @@ export class ProgrammingExerciseInstructionAnalysisService {
      * @param analysis arbitrary number of analysis objects to be merged into one.
      */
     private mergeAnalysis = (...analysis: Array<AnalysisItem[]>) => {
-        return compose(
```

Comment:
The substitute for compose would te `flowRight` in `lodash-es`.
Just in case you want to keep using it.

Reply:
I intentionally replaced lodash functions by built-in ones if possible, as I think that those are probably better tested. Additionally, replacing a function by an alias for it with probably the same implementation in the background would probably not fix a memory leak if it is caused there.

Reply:
It's not just an alias, the lodash-es actually fixes the memory leaks.
But I looked again and it makes sense here what you did.